### ISSUE TITLE
not working on realme gt neo 2 android 12 and request  support for taiwan ver.

### ISSUE THREAD

**Issue by wangdog123** [Kimjio/umamusume-localify-android#3]
Could you make this plugin support taiwan version?
it's painful for me to play on 30fps :(

And this plugin isnt working on my realme gt neo2 android 12(jp ver,
But i tried it on android emu and it worked.

**Comment by Kimjio** [comment]
If you check Logcat, do you see the log like this?

![Logcat img](https://user-images.githubusercontent.com/10695980/184672608-e7d7f7c4-b287-4072-b63a-155c7613dbc8.png)

If 'UmamusumeLocalify' is not displayed in the log, I think it is not applied.

**Comment by wangdog123** [comment]
> Owner

Hi,I just asked a person on discord eariler and after I tried the other plugin which uses lsposed, I just discovered that it didnt work cuz i turned on the denylists in magisk and add umamusume jp in it, that's why it didnt work.(because the plugin on lsposed said my umamusume is on deynlist so it might not work.
![image](https://user-images.githubusercontent.com/111276872/184680700-ea8619d7-643a-4f7a-a5ed-413d9928a55c.png)

After I turned off deynlist,
BOOM,IT WORKED!

The way I solved the denylist problem is to use this version of magisk https://huskydg.github.io/magisk-files/
It has the original magiskhide so I can play the game normally without setting the denylists.
But only runs with 60fps instead of higher although I set it to 120fps, anyways the game is a lot smoother than before,I love it!
